modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -130,6 +130,22 @@ def icn
         @current_user&.icn
       end
 
+      def handle_210966_data
+        roles = {
+          'fiduciary' => 'Fiduciary',
+          'officer' => 'Veteran Service Officer',
+          'alternate' => 'Alternate Signer'
+        }
+        data = JSON.parse(params.to_json)
+        if data['third_party_preparer_role']
+          data['third_party_preparer_role'] = (
+            roles[data['third_party_preparer_role']] || data['other_third_party_preparer_role']
```

Comment:
is it intentional that this first one is looking at "roles", but the second one is not?

Reply:
Yes, in cases where the role is one of our given options we want to format the string nicely (ie, refer to `roles`) but if they just write in an "other" option, we just translate directly what they wrote since we can't anticipate how to capitalize/spell it correctly.